src/log-manager/src/nginx/list_logs.lua

```diff
@@ -43,19 +42,19 @@ end
 
 local log_query_param = "?username="..username.."&framework-name="..framework_name..
   "&pod-uid="..pod_uid.."&taskrole="..taskrole.."&token="..token
-local path = "/usr/local/pai/logs/"..username.."/".. framework_name.."/".. taskrole.."/"..pod_uid.."/"
+local log_dir = "/usr/local/pai/logs/"..username.."/".. framework_name.."/".. taskrole.."/"..pod_uid.."/"
```

Comment:
To prevent the directory traversal attack, normalizing all user inputs before concatenating them into the real path.

Reply:
Fixed